Source/AliveLibAO/Slog.cpp

```diff
@@ -1840,8 +1841,41 @@ __int16 Slog::Brain_2_ChasingAbe_470F50()
 
 __int16 Slog::Brain_3_Dead_4721B0()
 {
-    NOT_IMPLEMENTED();
-    return 0;
+    if (field_14C)
+    {
+        field_14C->field_C_refCount--;
+        field_14C = 0;
+    }
+
+    if (field_10C)
+    {
+        field_10C->field_C_refCount--;
+        field_10C = 0;
+    }
+
+    if (field_11C < static_cast<int>(gnFrameCount_507670) + 80)
+    {
+        field_C0_r -= 2;
+        field_C2_g -= 2;
+        field_C4_b -= 2;
+        field_BC_sprite_scale -= FP_FromDouble(0.023);
```

Comment:
Check the other death brains, I think they use 0.3 ?

Reply:
nah pretty much all death brains have unique values